### PR TITLE
Send caught render error to sentry

### LINKED ISSUE #2455: Log catched render error to sentry
Related to: #2453
Originally implemented in: #598

Podla toho gistu tam chyba:

```js

if (typeof Raven !== 'undefined' && typeof Raven.captureException === 'function') {
		Raven.captureException(new Error(errorMsg), {
			extra: {
				errorStack: error.stack
			}
		});
	}
```

### PR DESCRIPTION
**Pull request by ujovlado**
Fixes #2455 
Related to #2666 

Proposed changes:

- Log caught render error to sentry

### CODE REVIEW

**Comment by ujovlado** [comment]
Testnut sa to da tak, ze si includnes do `index.html` raven

```html
<script src="https://assets-cdn.keboola.com/misc-ui/raven-js/3.17.0/raven.min.js"></script>
```

Initnes ho:

```js
Raven
    .config('zober dsn z test-project v sentry')
    .install();
```

A povol zareportovanie pre dev mod.

**Comment by ujovlado** [comment]
Dufam, ze po nasadeni nam to nezaplni ui-errors :)

File: src/scripts/utils/ReactErrorHandler.js [hunk 1]
```diff
@@ -26,6 +26,15 @@ function logError(Component, error) {
   const errorMsg = `Error. Check render() method of component '${Component.displayName || Component.name || '[unidentified]'}'.`;
 
   console.error(errorMsg, 'Error details:', error); // eslint-disable-line
+
+  /* global Raven */
+  if (!process.env.__DEV__ && typeof Raven !== 'undefined' && typeof Raven.captureException === 'function') {
```
Comment (kacurez) [inline src/scripts/utils/ReactErrorHandler.js]: nedalo by sa to `typeof Raven !== 'undefined'` na iba `Raven`?

Comment (ujovlado) [reply src/scripts/utils/ReactErrorHandler.js]: asi jo, ale nechcel som to menit oproti povodnemu kodu (co je v tom giste)

Comment (kacurez) [reply src/scripts/utils/ReactErrorHandler.js]: aha ok kludne to nechaj tak ako je.

**Comment by ujovlado** [comment]
ok, poslem to tam

**Pull request merged**